pkg/security/security_profile/dump/activity_dump.go

```diff
@@ -366,11 +366,13 @@ func (ad *ActivityDump) enable() error {
 
 	if len(ad.Metadata.ContainerID) > 0 {
 		// insert container ID in traced_cgroups map (it might already exist, do not update in that case)
-		if err := ad.adm.tracedCgroupsMap.Update(ad.Metadata.ContainerID, ad.LoadConfigCookie, ebpf.UpdateNoExist); err != nil {
+		containerID := make([]byte, 64)
+		copy(containerID, []byte(ad.Metadata.ContainerID))
```

Comment:
I'm curious why do we have to make a copy here ? is it related to this PR or is it an issue that we already had

Reply:
These is an artifact of a previous version of the PR. Removed it